test/e2e/auth/selfsubjectreviews.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	"github.com/onsi/ginkgo/v2"
 	"github.com/onsi/gomega"
+	authenticationv1 "k8s.io/api/authentication/v1"
 	authenticationv1alpha1 "k8s.io/api/authentication/v1alpha1"
 	authenticationv1beta1 "k8s.io/api/authentication/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
the e2e test should drop the `[Feature:APISelfSubjectReview]` tag so it actually runs in the main e2e job (and may need to make the alpha/beta portions conditional on availability of those APIs)

Reply:
To not forget: also required to delete it from the list here https://github.com/kubernetes/test-infra/blob/a66784ee843eeaa97a7a13b756301d1e9a2c91ab/config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml#L363